### PR TITLE
Helm Chart

### PR DESCRIPTION
**Pull request by shanesiebken**
*Is this a bug fix or adding new feature?*
New Feature.

Initial Helm Chart for driver components.

*What is this PR about? / Why do we need it?*
Starting point for work on #99

New PR because I'm fighting with git to get CLA signed on my account email address. Sorry for the churn :-1: 

*What testing is done?*
Deployed in staging cluster and confirmed output. More testing should be added.

More work should likely be done on this - but it captures a good portion of the "default" configuration of the driver components, as well as some of the more advanced functionality found in other charts - affinity, selectors, etc.

### CODE REVIEW

**Comment by leakingtapan** [comment]
@shanesiebken we are refactoring the manifest to align with recent upstream change. Please refer https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/171 This will need updated once the PR is closed

**Comment by shanesiebken** [comment]
No problem @leakingtapan. Sorry for the delay on this - I haven't had time to figure out how in the world to contact the team at CNCF, as my login / account on their site is totally messed up.

**Comment by leakingtapan** [comment]
@shanesiebken have you tried emailing: helpdesk@rt.linuxfoundation.org

**Comment by shanesiebken** [comment]
I haven't - I'll try that first thing in the morning! Thanks for the suggestion!

**Comment by leakingtapan** [comment]
／retest

**Comment by shanesiebken** [comment]
@leakingtapan Got my CLA signed! Not sure if there's anything more I need to do on here to get this PR to update, but let me know if there is.

**Comment by leakingtapan** [comment]
Our features have been involved a lot lately. Reviewing the code again for new changes

File: charts/aws-ebs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for the AWS EBS CSI Driver (https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+name: aws-ebs-csi-driver
+version: 0.1.0
```
Comment (leakingtapan) [inline charts/aws-ebs-csi-driver/Chart.yaml]: Update to 0.2.0 since it is the latest.

File: charts/aws-ebs-csi-driver/templates/attacher.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
+apiVersion: v1
```
Comment (leakingtapan) [inline charts/aws-ebs-csi-driver/templates/attacher.yaml]: we combined attacher and provisioner manifest few weeks back. Could you combine helm template as well? 

There are a bunches of changes made along with the combine. I think its easier to create new helm templates referring the existing manifests

File: charts/aws-ebs-csi-driver/values.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  #     - matchExpressions:
+  #       - key: key
+  #         operator: In
+  #         values:
+  #         - value
+
+node:
+  image:
+    repository: quay.io/k8scsi/driver-registrar
+    tag: v1.0-canary
```
Comment (leakingtapan) [inline charts/aws-ebs-csi-driver/values.yaml]: Should use v1.0.1 for all the sidecars

**Comment by leakingtapan** [comment]
/close

@shanesiebken please reopen if you got a chance to finish the comment

**Pull request closed without merging**